.jenkins

```diff
@@ -343,7 +343,7 @@ pipeline {
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=$WORKSPACE/bin/nvcc_wrapper \
-                                -DCMAKE_CXX_FLAGS=-Werror \
+                                -DCMAKE_CXX_FLAGS=-Werror --Werror=all-warnings -Xcudafe --diag_suppress=3159 \
```

Comment:
This suppresses the `long double` is `double` warning that we already working around with `NVHPC` in the `Complex` test.

Reply:
I am not very comfortable disabling altogether.  Did you consider disabling for `Complex` usage only?
Admittedly I don't quite remember if it is sufficient to add pragmas push/pop around the definition of the template class/functions or if you still get the warning at instantiation point.

Reply:
I don't think that we can easily disable it for the `Complex` test only, see https://stackoverflow.com/questions/68025764/whats-the-nvcc-warning-token-for-long-double-is-treated-as-double-in-devi.

Reply:
I suppose we will still see the `long double` warnings that can/should be fixed in other NVCC builds